### PR TITLE
Fix dependency for connect-datadog

### PR DESCRIPTION
**Pull request by vanessayuenn**
Dependency has switched from `node-dogstatsd` to `hot-shots` as of v0.0.9.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DataDog/node-connect-datadog/pull/11
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.
- [x] depends on https://github.com/microsoft/DefinitelyTyped-tools/pull/61 for tests to pass

### CODE REVIEW

**Comment by vanessayuenn** [comment]
Sorry for all the pings, folks 😓. Looks like my rebase as git default confused GitHub and requested reviews from a whole bunch of codeowners. My apologies.

Not sure how to proceed with this PR here, though. Looks like [CI is complaining](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45510/checks#step:5:100) about the version in the header, even though the version `0.0.9` does match the [current version](https://github.com/DataDog/node-connect-datadog/releases/tag/v0.0.9) of connect-dog I am making this fix for. And if I leave the version as is, CI _also_ yells at me for not changing the header.

File: types/connect-datadog/index.d.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@
 // TypeScript Version: 2.3
 
 import express = require('express');
-import dogstatsd = require('node-dogstatsd');
+import dogstatsd = require('hot-shots');
```
Comment (lewisvail3) [inline types/connect-datadog/index.d.ts:10]: Same as above.
```suggestion
import hotShots = require('hot-shots');
```

**Review by lewisvail3 — CHANGES_REQUESTED** [review]

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## wavesurfer.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/wavesurfer.js@latest/))
was missing the following properties:
1. debounce
1. ajax
1. getId
1. max
1. min

<details>
<summary>as well as these 8 other properties...</summary>
<p>Observer, extend, style, requestAnimationFrame, frame, preventClick, fetchFile, WaveSurfer</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e7fba5cd1ba760e811e9092aac50ac43fd893014
</p>

**Review by lewisvail3 — APPROVED** [review]

**Review by Daddy2200 — COMMENTED** [review]
@vanessayuenn @typescript-bot @danger-public @lewisvail3 @Jessidhia @orta @elibarzilay @alloy @alexeagle

**Comment by Daddy2200** [comment]
**_[****](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45510#issue-435075268)_**

**Pull request merged**